requirements.txt

```diff
@@ -1,7 +1,7 @@
 virtualenv==16.7.10
 asv==0.4.1
 gitpython
-dvc==0.94.0
+dvc[s3]==0.94.0
```

Comment:
Aah, right, here is the answer to that run-cache question. We are still using 0.94.0 here. Ok, let's not switch to 1.0 for now. We'll do that after the official realeas.